src/app/core/services/error-message.service.ts

```diff
@@ -8,8 +8,12 @@ import { Injectable } from '@angular/core';
  * Contains all error message prompts to user.
  */
 export class ErrorMessageService {
+  public static repoPrefixNotPresentMessage() {
+    return 'Invalid repo prefix. Provide a repository URL or repo name with a valid prefix. (e.g. <ORG/USER>/<REPO>)';
```

Comment:
Hmm.. I thought it might be too clunky to name it `organisationOrUserNotPresentMessage`, thats why i tried using "repo prefix" instead, but I understand that the term may not be intuitive enough

Do you have any suggestions as to what we can name it? I was thinking maybe `repoOwnerNotPresentMessage`?

Reply:
`repoOwnerNotPresentMessage` sounds good.